content/guides/cloud/smart-orchestration.md

```diff
@@ -42,41 +41,47 @@ time.**
 
 </Alert>
 
-### Toggle running of failed specs first
+### Activate Spec Prioritization in Cypress Cloud
 
 <Alert type="success">
 
 <strong class="alert-header"><Icon name="star"></Icon> Premium Cypress Cloud
 Feature</strong>
 
-**Running failed specs first** is a _Smart Orchestration_ feature available to
-users with a [Business Cypress Cloud plan](https://cypress.io/pricing).
+**Spec Prioritization** is a _Smart Orchestration_ feature available to users
+with a
+[Business or Enterprise tier Cypress Cloud plan](https://cypress.io/pricing).
+Wondering if upgrading your Team tier subscription is worth it? Check out the
+[recommendations](/guides/cloud/runs#Recommendations) on the run overview page
+to see how much time we predict these features would save you.
 
 </Alert>
 
-Controlling running of failed specs first is a _Smart Orchestration_ feature
-that is managed within a project's settings.
+Spec Prioritization is a _Smart Orchestration_ feature that is managed within a
+project's settings.
 
 To enable or disable this feature:
```

Comment:
```suggestion
To enable or disable this feature at the Project level (must be an admin user):
```
@debrisapron this was a last minute change, which also led to a postpone of our release to Monday.